src/platform/plugins/shared/dashboard/server/content_management/v1/cm_services.ts

```diff
@@ -317,6 +317,14 @@ export const dashboardItemSchema = schema.object(
     references: schema.arrayOf(referenceSchema),
     namespaces: schema.maybe(schema.arrayOf(schema.string())),
     originId: schema.maybe(schema.string()),
+    accessControl: schema.maybe(
+      schema.object({
+        owner: schema.maybe(schema.string()),
```

Comment:
How is `owner` field different then existing `createdBy` field? How come `owner` field can is provided in read but can not be set during write?